lib/driver.js

```diff
@@ -1227,6 +1227,11 @@ class XCUITestDriver extends BaseDriver {
     const shouldGetDeviceCaps = _.isBoolean(this.opts.includeDeviceCapsToSessionInfo)
       ? this.opts.includeDeviceCapsToSessionInfo
       : true; // Backward compatibility
+    if (shouldGetDeviceCaps) {
+      log.warn(`Please call 'mobile:deviceScreenInfo' to get 'pixelRatio' and 'statBarHeight', ` +
```

Comment:
I would rather remove this warning as it is the default behaviour, so it could be confusing for users

Reply:
ok. i just wondered if we addressed this point...